src/screens/OtherProfile/index.js

```diff
@@ -566,15 +427,12 @@ const OtherProfile = () => {
           isAnonimity={isAnonimity}
           bio={dataMain.bio}
           openBio={openBio}
+          dataMain={dataMain}
           isSignedMessageEnabled={isSignedMessageEnabled}
           showSignedMessageDisableToast={showSignedMessageDisableToast}
           loadingGenerateAnon={loadingGenerateAnon}
           avatarUrl={profile.myProfile.profile_pic_path}
           anonProfile={anonProfile}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The props `onSendDM`, `setDMChat`, `loadingSendDM`, and `dmChat` are removed from the `BioAndChat` component. Ensure that the functionality provided by these props is either no longer required or is being handled within the `BioAndChat` component itself.
```diff
- onSendDM={onSendDM}
- setDMChat={setDMChat}
- loadingSendDM={loadingSendDM}
- dmChat={dmChat}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
ok